doc/classes/Area3D.xml

```diff
@@ -89,7 +89,8 @@
 	</methods>
 	<members>
 		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp" default="0.1">
-			The rate at which objects stop spinning in this area. Represents the angular velocity lost per second. Values range from [code]0[/code] (no damping) to [code]1[/code] (full damping).
+			The rate at which objects stop spinning in this area. Represents the angular velocity lost per second.
+			See [member ProjectSettings.physics/3d/default_linear_damp] for more details about damping.
```

Comment:
Maybe also copy the note to `default_angular_damp` and link it for the `angular_damp` properties? Currently it's the same text but the `default_angular_damp` property could use the extra documentation too, and if linear and angular end up being treated differently, we'll only have to change one place.